src/qml/AboutPage.qml

```diff
@@ -29,6 +29,28 @@ Flickable {
         id: diskUsage
     }
 
+    Item {
+        id: uptimeCounter
+        property int totaltime : uptime
+
+        Timer {
+            interval: 1000
+            repeat: true
+            triggeredOnStart: true
+            running: true
+            onTriggered: ++uptimeCounter.totaltime
```

Comment:
Yes, it's absolutely possible for it to be off by up to the duration of time that the user keeps the Settings app open.  For example, if you open the About page, watch the uptime for 30 seconds, then go back to the Settings page, and re-open the About page, it will appear to have lost those 30 seconds.  I did it this way for simplicity (only 6 additional lines of C++) and because I suspect it doesn't matter much in real life.

The alternative would be to add a header file, another C++ file and create a QML object and register it in `main.cpp`. Then somebody would likely suggest that we make it a general purpose object and... it just didn't seem worth that much.

Reply:
Rewrote everything.